packages/flutter/lib/src/painting/image_stream.dart

```diff
@@ -489,6 +491,13 @@ abstract class ImageStreamCompleter with Diagnosticable {
   /// if all [keepAlive] handles get disposed.
   bool _hadAtLeastOneListener = false;
 
+  /// Whether the listeners were added before a completer was attached
```

Comment:
```suggestion
  /// Whether the future listeners added to this completer are initial listeners.
  ///
  /// This can be set to true when a [ImageStream] adds its initial listeners to this completer.
```